x/concentrated-liquidity/model/pool.go

```diff
@@ -50,10 +45,9 @@ func NewConcentratedLiquidityPool(poolId uint64, denom0, denom1 string, tickSpac
 		Token0:               denom0,
 		Token1:               denom1,
 		TickSpacing:          tickSpacing,
-		ExponentAtPriceOne:   exponentAtPriceOne,
+		ExponentAtPriceOne:   sdk.NewInt(-6),
```

Comment:
Hardcoding here for now, in the event we are looking to change to a different exponent / tick spacing pair.